src/render.rs

```diff
@@ -229,6 +229,16 @@ impl Render for Filter {
                     }
                 }
             }
+            FilterType::AddSlashes => match left {
+                Some(content) => Some(Content::String(Cow::Owned(
+                    content
+                        .render(context)?
+                        .replace("\\", "\\\\")
+                        .replace("\"", "\\\"")
```

Comment:
```suggestion
                        .replace(r#"""#, r#"\""#)
```

Though I'm not entirely sure this one is easier to read.

Reply:
I left this as is, as it didn't seem that much more readable to me.